### PR TITLE
Replace boot2docker with docker-machine (Toolbox)

### PR DESCRIPTION
**Pull request by mbonaci**

### CODE REVIEW

**Comment by tamird** [comment]
Thanks for the PR! Unfortunately the current VirtualBox release that ships with Docker Toolbox suffers from [this bug](https://www.virtualbox.org/ticket/14412) which makes Docker Toolbox a non-starter.

Could you base this PR to remove the merge commit? I'll then merge this when a new release of VirtualBox/Docker Toolbox is available.

File: README.md [hunk 1]
```diff
@@ -51,14 +51,16 @@ FATA[0000] Post http:///var/run/docker.sock/v1.17/images/create?fromImage=cockro
 ```
 On OSX:
 ```bash
-# Setup Boot2Docker. This should only need to be done once.
-boot2docker init
```
Comment (tamird) [inline README.md]: The equivalent docker-machine command is `docker-machine start default`

File: README.md [hunk 2]
```diff
@@ -51,14 +51,16 @@ FATA[0000] Post http:///var/run/docker.sock/v1.17/images/create?fromImage=cockro
...
-# Start Boot2Docker. This will need to be run once per reboot.
-boot2docker start
```
Comment (tamird) [inline README.md]: The equivalent docker-machine command is `docker-machine start default`, we should still mention it.

File: README.md [hunk 3]
```diff
@@ -51,14 +51,16 @@ FATA[0000] Post http:///var/run/docker.sock/v1.17/images/create?fromImage=cockro
...
-# Setup environment variables. This will need to be run once per shell.
-$(boot2docker shellinit)
+# After installing [Docker Toolbox](https://www.docker.com/toolbox),
```
Comment (tamird) [inline README.md]: these instructions are already in the context of the person running their own shell and running into the ssl issue. I think all we need is to:
- make sure the vm is running: `docker-machine start default`
- initialize the shell: `eval $(docker-machine env default)`

**Comment by mbonaci** [comment]
Yeah, there's a workaround [here](https://github.com/docker/machine/issues/1498), basically VBox downgrade.

> rebase

**Pull request closed without merging**